server/repository/src/migrations/v1_06_00/mod.rs

```diff
@@ -41,6 +42,7 @@ impl Migration for V1_06_00 {
         master_list::migrate(connection)?;
         temperature_breach::migrate(connection)?;
         patient_id_indices::migrate(connection)?;
+        indexes::migrate(connection)?;
```

Comment:
Moved this up so that the indexes will be created before any merging, so that indexes don't get created after a column has been dropped... probably wouldn't matter once merging stuff is moved to 1.7 though..